### PR TITLE
doc(ui maintenance) ui maintenance docs CFY-5743

### PR DESCRIPTION
**Pull request by edenpessach**

### CODE REVIEW

File: content/manager_webui/maintenance.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: bt_wiki
+title: Maintenance Mode
+category: Manager
+draft: false
+abstract: Cloudify's Web Interface
+weight: 200
+---
+
+Managing maintenance mode page will be found at the settings section in the Web UI. Upon routing you will be presented with the manager maintenance mode status and a button used to change it.<br/>
```
Comment (ChenRoth) [inline content/manager_webui/maintenance.md]: replace with `The Maintenance Mode page is found in the settings section, where you may see and toggle the manager's maintenance mode.`

File: content/manager_webui/maintenance.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+![maintenance mode page]({{< img "ui/maintenance/ui-maintenance-mode-page.png" >}})
+
+
+Pressing the button to change the maintenance mode status opens a confirmation dialog.<br/>
+![confirmation dialog]({{< img "ui/maintenance/ui-maintenance-confirmation-dialog.png" >}})
+
+
+Confirming makes the manager start entering requested mode, a message will be shown notifying all users.<br/>
```
Comment (ChenRoth) [inline content/manager_webui/maintenance.md]: On confirmation, the manager enters the requested mode and a message is shown to any user currently using the UI.

File: content/manager_webui/maintenance.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+![starting maintenance message]({{< img "ui/maintenance/ui-starting-maintenance-message.png" >}})
+
+
+Once maintenance mode is turned on the Web UI will prevent all actions but turning maintenance mode off.<br/>
```
Comment (ChenRoth) [inline content/manager_webui/maintenance.md]: Once maintenace mode is turned on, only the maintenance mode page is available.

File: content/manager_webui/maintenance.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+![maintenance message]({{< img "ui/maintenance/ui-maintenance-message.png" >}})
+
+
+The manager will not enter maintenance mode until all running executions are finished. When requesting to turn on maintenance mode while other executions are running, you will be presented with the executions details and an option to cancel each.<br/>
```
Comment (ChenRoth) [inline content/manager_webui/maintenance.md]: When entering Maintenance Mode, the manager is waiting for all running executions to finish first.
During this time, you may view all running executions and even cancel them, if necessary.

File: content/manager_webui/maintenance.md (hunk 2 above)
Comment (edenpessach) [inline content/manager_webui/maintenance.md]: It doesn't enter said mode on confirmation, only checks if it can.

Comment (ChenRoth) [inline content/manager_webui/maintenance.md]: then write "the manager is trying to enter the requested mode"

**Pull request merged**